config/samples/nvidia_v1alpha1_nvidiadriver.yaml

```diff
@@ -18,6 +18,8 @@ spec:
     useHostMofed: false
   gds:
     enabled: false
+  persist: true
```

Comment:
we don't want to enable this by default but only when CDI is enabled. Need to handle this in the code to error out accordingly.